tests/waku_rln_relay/test_rln_group_manager_onchain.nim

```diff
@@ -775,7 +779,27 @@ suite "Onchain group manager":
       isReady == true
 
     await manager.stop()
+  
+  asyncTest "rln-relay-max-message-limit testing":
```

Comment:
i am not sure if this test belongs here since we are just testing the behaviour of the onchain group manager in this module
can you add a todo comment mentioning it should be moved later?